cves/CVE-2017-17081.yml

```diff
@@ -64,7 +64,14 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  NOTE: This third file was accidentally started - disregard (or don't if you're feeling extra credit-y?)
+
+  The library does not properly authetnicate widths and heights of a passed-in
+  MPEG stream. An attacker can utilize this to craft a video which has actual
+  width/heights (in-content) less than specified (causing a signedness error),
```

Comment:
Can you go deeper into "signedness"?

Reply:
More of a miswording really, it's an integer underflow from a misformed bounds check. roughly/off-memory: `if ((unsigned) itrX >= width - inputW)`, where `width < inputW` caused a comparison between (unsigned) `itrX` and (signed, yet negative) right-hand expression.